test/build.spec.js

```diff
@@ -0,0 +1,11 @@
+const Maker = require('../dist/src');
+
+async function createMaker(privateKey) {
+  return Maker.create('test', { privateKey, log: false });
+}
+
+test('can get a service from maker from the dist folder', async () => {
+  const maker = await createMaker();
+  const ethCdp = maker.service('cdp');
+  expect(!!ethCdp).toBe(true);
```

Comment:
since you're going to the trouble of spinning up the test chain you may as well do a simple "open cdp, lock eth, draw dai" test